src/emulator/functionsEmulator.ts

```diff
@@ -424,7 +424,7 @@ export class FunctionsEmulator implements EmulatorInstance {
               added = await this.addPubsubTrigger(this.args.projectId, definition);
               break;
             case Constants.SERVICE_AUTH:
-              added = await this.addAuthTrigger(this.args.projectId, definition);
+              added = this.addAuthTrigger(this.args.projectId, definition);
```

Comment:
This one is a bit suspicious to me.  I guess because auth is in-process it can be added synchronously?

Reply:
yup - it's a sync function, not async, so no need to await